src/SFA.DAS.Employer.Profiles.Web.UnitTests/Models/WhenBuildingAddUserDetailsViewModel.cs

```diff
@@ -0,0 +1,22 @@
+using FluentAssertions;
+using SFA.DAS.Employer.Profiles.Web.Models;
+
+namespace SFA.DAS.Employer.Profiles.Web.UnitTests.Models;
+
+public class WhenBuildingAddUserDetailsViewModel
```

Comment:
might be worth adding tests for the other links in the view model

Reply:
Updated the PR with the recommended changes with more unit testing coverage.
Areas covered 
1) RedirectUrl link
2) Error Dictionary in case of ModelState Error